docs/generators/java.md

```diff
@@ -100,7 +100,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |useRuntimeException|Use RuntimeException instead of Exception. Only jersey2, jersey3, okhttp-gson, vertx, microprofile support this option.| |false|
 |useRxJava2|Whether to use the RxJava2 adapter with the retrofit2 library. IMPORTANT: This option has been deprecated.| |false|
 |useRxJava3|Whether to use the RxJava3 adapter with the retrofit2 library. IMPORTANT: This option has been deprecated.| |false|
-|useSingleRequestParameter|Setting this property to true will generate functions with a single argument containing all API endpoint parameters instead of one argument per parameter. ONLY jersey2, jersey3, okhttp-gson, microprofile, Spring RestClient, Spring WebClient support this option.| |false|
+|useSingleRequestParameter|Setting this property to true will generate functions with a single argument containing all API endpoint parameters instead of one argument per parameter. ONLY jersey2, jersey3, okhttp-gson, microprofile, Spring RestClient, Spring WebClient support this option.| |null|
```

Comment:
Please make the default value `"false"`, so that it's clear for users migrating from older OpenAPI Generator versions that the default doesn't change.

Reply:
I guess this markdown file is being generated after executing one of these commands:

```bash
./bin/utils/export_docs_generators.sh || exit 
./bin/generate-samples.sh ./bin/configs/*.yaml || exit
```

and I also assume the data type in the markdown is being generated based on the Java type, since we changed it from `boolean` to `String` is being inferred as `null`.

Do you know where should I override this? @martin-mfg

Reply:
Your assumptions are correct.
You can simply merge [this commit](https://github.com/martin-mfg/openapi-generator/commit/0f5e01d5d0d4042bfd111fc4d1740015397eb21e), or repeat the shown changes manually in your branch.

Reply:
Thanks, addressed with [bd2c149](https://github.com/OpenAPITools/openapi-generator/pull/20590/commits/bd2c1492eba4a4e527bd7830d7734fc27e91f375)